directory-manifest/pomx.xml

```diff
@@ -0,0 +1,65 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>gov.nist.secauto.oscal.tools.core</groupId>
+    <artifactId>ci-cd-deps</artifactId>
+    <version>1.0.0</version>
```

Comment:
Minor, non-blocking: I would correct this to denote it is metaschema, not a strictly OSCAL project POM file.

```suggestion
    <groupId>gov.nist.secauto</groupId>
    <artifactId>metaschema-xslt</artifactId>
    <version>1.0.0</version>
```

Blocking (maybe): as asked in a prior comment, is this file intentionally named incorrectly?

Reply:
Actually we're pulling the file, as it was meant to serve as an example of brokenness, for testing.